assets/html/documenter.css

```diff
@@ -21,7 +21,8 @@ body, input {
 }
 
 pre, code {
-  font-family: 'Ubuntu Mono', Monaco, courier, monospace;
+  font-family: 'Source Code Pro', Monaco, courier, monospace;
+  font-size: 0.95em;
```

Comment:
I noticed that there should be an additional
```css
pre code {
  font-size: 1em;
}
```
selector as well. This is because we have `<pre><code>` blocks, and so the `.95` would get applied twice, yielding `.90` fonts in code blocks.